src/containers/PageContainer/index.tsx

```diff
@@ -93,23 +107,37 @@ const PageContainer: React.FC = ({ children }) => {
 				{/* Toast Notifications */}
 				<NotificationDrawer />
 
-				{/* App Header */}
-				<Header
-					pageWidth={pageWidth}
-					znsDomain={znsDomain}
-					domainMetadata={domainMetadata}
-					account={account}
-					openModal={openModal}
-				/>
-
-				{/* App Sidebar */}
-				<SideBar />
-
 				{/* App level Modals */}
 				<Modals pageWidth={pageWidth} modal={modal} closeModal={closeModal} />
 
-				{/* Children Components */}
-				<main className="main">{children}</main>
+				<div className={styles.InnerContainer}>
+					<div className={styles.FlexRowWrapper}>
+						{/* App Sidebar */}
+						<SideBar />
+						<div className={styles.FlexColumnWrapper}>
+							{/* App Header */}
+							<Header
+								pageWidth={pageWidth}
+								znsDomain={znsDomain}
+								domainMetadata={domainMetadata}
+								account={account}
+								openModal={openModal}
+								isScrollDetectionDown={isScrollDetectionDown}
+							/>
+
+							{/* Children Components */}
+							<main className={styles.Main}>{children}</main>
+						</div>
+						{/* Actions */}
+						<Actions
```

Comment:
Re-added actions here. When reaching laptop/desktop breakpoint in the figma designs, the actions are removed from the header and are required outside of that container...tricky one.